src/controllers/investmentController.ts

```
@@ -155,7 +157,7 @@ export async function getInvestmentWithdrawRequests(
     const nextCursor = hasMore ? page[page.length - 1].id : null;
 
     res.status(200).json({
-      requests: page.map((r: InvestmentWithdrawalRequest) => ({
+      requests: page.map((r: typeof list[number]) => ({
```

ESLint

[error] Replace `typeof·list` with `(typeof·list)`

(prettier/prettier)